data/sensors_recent.php

```diff
@@ -222,6 +273,51 @@ function output_cell($rowspan, $data) {
 			$rowspan = 1;
 		}
 
+		// enhance gwdata with alias and geo
+		$gateways2 = array();
+		foreach ($gateways as $gwdata) {
+			if ($deb > 1) {
+				print("310 ");
+				print_r($gwdata);
+				print("<br>");
+			}
+			// add alias if missing
+			if (! isset($gwdata['alias']))
+				$gtw_id = $gwdata['gtw_id'];
+				if (isset($ttn_gw_data[$gtw_id]['alias'])) 
+					$gwdata['alias'] = $ttn_gw_data[$gtw_id]['alias'];
```

Comment:
Deze regels hebben de verkeerde indentation (of er mist een `{` en `}` als deze if binnen de if erboven hoort.

Daarnaast: zijn deze twee regels uberhaupt nodig? Volgens mij staan ze hieronder (regel 293/294) nog een keer identiek, en zijn ze (binnen de `if(!unknown)` beter op hun plek.